package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
So maybe we could use the extension feature of `isbinaryfile` to avoid reading file if it has a known extension ?

@see https://github.com/yeoman/generator/pull/737#issuecomment-70362751